examples/standalone_single_project_confidential_nodes/README.md

```diff
@@ -0,0 +1,146 @@
+- multiclusterservicediscovery.googleapis.com
+- networkmanagement.googleapis.com
+- secretmanager.googleapis.com
+- servicemanagement.googleapis.com
+- servicenetworking.googleapis.com
+- serviceusage.googleapis.com
+- sourcerepo.googleapis.com
+- sqladmin.googleapis.com
+- storage-api.googleapis.com
+- trafficdirector.googleapis.com
+
+The entity used to deploy this examples must have the following roles at Project level:
+
+- Artifact Registry Admin: `roles/artifactregistry.admin`
+- Certificate Manager Owner: `roles/certificatemanager.owner`
+- Cloud Build Builder: `roles/cloudbuild.builds.builder`
+- Cloud Build Worker Pool Owner: `roles/cloudbuild.workerPoolOwner`
+- Cloud Deploy Service Agent: `roles/clouddeploy.serviceAgent`
+- Cloud Deploy Admin: `roles/clouddeploy.admin`
+- Compute Admin: `roles/compute.admin`
+- Network Admin: `roles/compute.networkAdmin `
+- Security Admin: `roles/compute.securityAdmin`
+- Container Admin: `roles/container.admin  `
+- Cluster Admin: `roles/container.clusterAdmin`
+- DNS Admin: `roles/dns.admin`
+- GKE Hub Admin: `roles/gkehub.editor`
+- GKE Hub Scope Admin: `roles/gkehub.scopeAdmin`
+- Service Account Admin: `roles/iam.serviceAccountAdmin`
+- Service Account User: `roles/iam.serviceAccountUser`
+- Logging LogWriter: `roles/logging.logWriter`
+- Project IAM Admin: `roles/resourcemanager.projectIamAdmin`
+- Service Usage Admin: `roles/serviceusage.serviceUsageAdmin`
+- Source Repository Admin: `roles/source.admin` (if using CSR)
+- Storage Admin: `roles/storage.admin`
+- Project AdminL `roles/resourcemanager.projectIamAdmin`
+- Viewer: `roles/viewer`
+
+The entity used to deploy this examples must have the following roles at Organization level:
+
+- Organization Administrator: `roles/resourcemanager.organizationAdmin`
+- Access Context Manager Policy Admin: `roles/accesscontextmanager.policyAdmin`
+
+This example requires a Single network configured:
+
+- One subnet for Cluster
+- One subnet for a NAT
+- DNS Policy With inbound fowarding enabled
+- A [VM Proxy machine configured for Private Worker Pool](https://cloud.google.com/build/docs/private-pools/access-external-resources-using-static-external-ip#access_external_resources_in_a_private_network)
+- [Private Service Connect Configured](https://cloud.google.com/build/docs/private-pools/using-vpc-service-controls)
+
+This examples also require a VPC-SC Perimeter created and [configured with project](https://cloud.google.com/vpc-service-controls/docs/set-up-service-perimeter).
+
+## Usage
+
+the steps below assume that you are checked out on the same level as `terraform-google-enterprise-application` directory:
+
+```txt
+.
+├── terraform-google-enterprise-application
+└── .
+```
+
+1. Enter at Single Project example folder:
+
+    ```bash
+    cd terraform-google-enterprise-application/examples/standalone_single_project
+    ```
+
+1. Update `terraform.tfvars`.
+
+1. Run `terraform plan` and check the information
+
+1. Run `terraform apply`.
```

Comment:
In 2-multitenant/envs, only the development environment specifies an AUTOPILOT cluster. For nonproduction and production, there is no specific setting for the cluster_type, and it takes the default value of STANDARD_NAP.
This means that the nonproduction and production will be created without confidential nodes (the env_baseline/main.tf applies confidential nodes only to autopilot clusters)